metricbeat/module/kubernetes/state_node/_meta/fields.yml

```diff
@@ -15,6 +15,22 @@
           type: boolean
           description: >
             Node unschedulable status
+        - name: memorypressure
+          type: boolean
+          description: >
+            Node MemoryPressure status
+        - name: diskpressure
+          type: boolean
+          description: >
+            Node DiskPressure status
+        - name: outofdisk
+          type: boolean
+          description: >
+            Node OutOfDisk status
+        - name: pidpressure
```

Comment:
Separate words with underscore in these metrics (`memory_pressure`, `disk_pressure`, `out_of_disk`, `pid_pressure`).

Reply:
@jsoriano Thanks for the review.

Sure, this makes sense.